### ISSUE TITLE
phonetisaurus scripts

### ISSUE THREAD

**Issue by jtrmal** [kaldi-asr/kaldi#1568]
The sequitur G2P is slow to train and not well maintained.
Until recently, Phonetisaurus was trickier to install (different openfst), but now when we switched to stock openfst, it should be relatively easy to streamline it.
@adolfvonkleist has agreed to create phonetisaurus scripts compatible with the sequitur scripts together. this is to gently remind him and keep track of the issue.

**Comment by jtrmal** [comment]
@danpovey I wrote some scripts myself. I think it could be quite straightforward to include (as an extras/ ) the build script as well. There is a quite significant interaction with kaldi build system -- it needs the openfst to be compiled with more options (more extensions). I didn't explore yet if all those extensions are necessary. Are you ok with building the kaldi openfst with more extensions? Or shall I compile another openfst from scratch? As far as I can tell, it should not have any other impact (apart from perhaps longer compile time of openfst, which might complicate things for Travis).

**Comment by AdolfVonKleist** [comment]
Hi, We don't necessarily need all of the extensions. In fact for the
current build it should be ok with the kaldi defaults - maybe with some
really slight tweaking.  The .far stuff is used for dumping alignment
lattices, but I don't think this has ever really been used, and can
probably be omitted.   I think we can get away with the extras - and
apologies for the delay here on my end.  If you can wait I can commit to
push it out this weekend for you to review. -joe

**Comment by jtrmal** [comment]
Hi Joe, sure, we can wait until Fr. BTW, I created two new PRs in github
phonetisauris repo, so if you could please review it?
Thanks,
y.

**Comment by AdolfVonKleist** [comment]
OK I've created a pull request for this here: 
- https://github.com/kaldi-asr/kaldi/pull/1734

Ok I've tried to tidy things up a bit in my g2p repository, and tied this to a ```kaldi``` tag there, with the idea at least that that should keep it stable against the ```install_phonetisaurus.sh``` script I've added.  I tried to follow/borrow/copy as much as possible from the ```install_sequitur.sh``` script.  

In this first attempt I've stuck to just the c++ binaries, and skip compiling the python binding by default here for simplicity, although I did retain the check for ```Python.h``` that is included in the sequitur script. 

Hopefully all on the right track.
